weblate/billing/models.py

```diff
@@ -256,9 +253,7 @@ def all_projects(self):
 
     @cached_property
     def projects_display(self):
-        return format_html_join_comma(
-            "{}", (x for x in list_to_tuples(self.all_projects))
-        )
+        return format_html_join_comma("{}", list_to_tuples(x for x in self.all_projects))
```

Comment:
These no-op expressions are not really needed (more of these are present):

```suggestion
        return format_html_join_comma("{}", list_to_tuples(self.all_projects))
```